### PR TITLE
Add FAQ for design library

### PR DESCRIPTION
**Pull request by gtsiap**
This is an FAQ for `DesignLibrary`.

### CODE REVIEW

File: TechnicalDocuments/DesignLibrary.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        appearance: BabylonAppAppearance,
+        config: Config
+    ) {
+        self.observer = observer
+        self.appearance = appearance
+        self.config = config
+    }
+```
+
+`BabylonAppAppearance` is `DesignLibrary.ComponentsBuilder` so you can call `appearance` in order to reuse Bento components, here is an example
```
Comment (inamiy) [inline TechnicalDocuments/DesignLibrary.md]: [Nit]

```suggestion
`BabylonAppAppearance` is a typealias for `DesignLibrary.ComponentsBuilder` so you can call `appearance` in order to reuse Bento components, here is an example
```

File: TechnicalDocuments/DesignLibrary.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# DesignLibrary Technical Documentation
+
+This is a FAQ for `DesignLibrary`.
+
+1. How can I reuse viewcontrollers?
```
Comment (ilyapuchka) [inline TechnicalDocuments/DesignLibrary.md]:
```suggestion
1. How can I reuse view controllers?
```

File: TechnicalDocuments/DesignLibrary.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+```swift
+DesignLibrary.viewControllersBuilder().navigationController(....)
+```
+
+2. How can I use `DesignLibrary`?
```
Comment (ilyapuchka) [inline TechnicalDocuments/DesignLibrary.md]: [nit] this should be probably the first item, it's the most important one.

**Comment by ilyapuchka** [comment]
@gtsiap this document should contain a link to the design library components (in Zeplin?)

**Pull request merged**